bucket/make.json

```
@@ -1,5 +1,6 @@
 {
     "homepage": "https://www.gnu.org/software/make/",
+    "description": "A tool which controls the generation of executables and other non-source files of a program from the program's source files.",
```

I think this could be enough, but feel free to skip it.

```suggestion
    "description": "Build automation tool.",
```